DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/projectIndexResults/metadata.json

```diff
@@ -0,0 +1,215 @@
+                "",
+                "* These indexed terms can later be queried on to get their matching documents, the documents that contain them.",
+                "",
+                "* In this example, the 'HiredAt.Year' data is indexed in the _WorkingInCompanySince_ index-field."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug" : "step-5",
+            "DescriptionLines": [
+                "* Query the 'Employees' collection using the defined index.",
+                "",
+                "* The 'Employees' documents can be _filtered_ by the index-fields defined in `Employees_ByWorkPeriod.IndexEntry`",
+                "",
+                "* The resulting documents will include only employees that started working after 'startYear'."
+            ],
+            "Assets": [
+                
+                {
+                    "Title": "Filtering Results - Basics Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/queries/filtering-results-basics"
+                }
+            ]
+        },
+        {
+            "Title": "Project Results",
+            "Slug" : "step-6",
+            "DescriptionLines": [
+                "* `selectFields()` must be used to satisfy the compiler and be able to select fields from the Employee class type,",
+                "  otherwise the compiler will only allow access to the IndexEntry class details.",
+                "",
+                "* Use `selectFields()` to select which specific document fields will be returned from the server for each resulting document.<br>",
+                "  Only these fields will be returned in the projected class `EmployeeProjectedDetails` and Not the whole document.",
+                "",
+                "* `QueryData.customFunction()` let us select the wanted fields and how they would be.",
+                "",
+                "* In this example:<br>",
+                "  * _FirstName_, _Phone_ are returned from the document<br>",
+                "  * _Location_ is customized from the other raw document fields."
```

Comment:
Lower case for firstName, phone & location

Reply:
Done

Reply:
What about 'Location' ? .....

Reply:
Done